File: SSServer/src/main/java/com/prose/crhen/SSServer/db/ServerRepository.java
package com.prose.crhen.SSServer.db;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Component;
import org.springframework.stereotype.Repository;

import com.prose.crhen.SSServer.model.Server;

@Repository
public interface ServerRepository extends CrudRepository<Server, Long> {

}

File: SSKotlinServer/src/main/kotlin/com/prose/crhen/SSKotlinServer/SSKotlinServer/db/ServerRepository.kt
package com.prose.crhen.SSKotlinServer.SSKotlinServer.db

import com.prose.crhen.SSKotlinServer.SSKotlinServer.model.Server
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ServerRepository: JpaRepository<Server, Long> {

    fun findByName(name: String): Server
}
